### PR TITLE
Remove 2019.2 support

### PR DESCRIPTION
**Pull request by hunterwerlla**
- Remove 2019.2 support, this is going into a feature branch because we are not doing this immediately
- On top off this will be a PR that fixes as many of the FIX_WHEN_MIN_IS_193 comments as possible

## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: jetbrains-rider/tst/software/aws/toolkits/jetbrains/services/lambda/completion/DotNetHandlerCompletionTest.kt [hunk 1]
```diff
@@ -36,10 +35,9 @@ class DotNetHandlerCompletionTest : BaseTestWithSolution() {
         assertIconPath(handlers.first().iconId, "/resharper/PsiSymbols/Method.svg")
     }
 
-    @Test
+    // TODO this test only works on 2019.2. Which we don't support anymore. Fix the test
```
Comment (kiiadi) [inline jetbrains-rider/tst/software/aws/toolkits/jetbrains/services/lambda/completion/DotNetHandlerCompletionTest.kt:38]: How do we get this test fixed?

File: jetbrains-rider/build.gradle [hunk 2]
```diff
@@ -69,20 +69,18 @@ tasks.withType(prepareSandbox.class).all {
 
 compileKotlin.dependsOn(generateModel)
 
-// TODO: Remove this FIX_WHEN_MIN_IS_193
-tasks.findByName("compileTestKotlin").onlyIf { resolveIdeProfileName() != "2019.2" /* Tests require 2019.3 or higher to build and run */ }
+tasks.findByName("compileTestKotlin")
```
[the remark this answers is not in the document]
Comment (hunterwerlla) [reply jetbrains-rider/build.gradle:72]: Ah, yes, I misinterpreted this line.

File: jetbrains-rider/build.gradle [hunk 3]
```diff
@@ -69,20 +69,18 @@ tasks.withType(prepareSandbox.class).all {
...
 test {
     systemProperty("log.dir", "${project.intellij.sandboxDirectory}-test/logs")
     useTestNG()
     environment("LOCAL_ENV_RUN", localEnv)
-}.onlyIf { resolveIdeProfileName() != "2019.2" /* Tests require 2019.3 or higher to build and run */ }
+}
 
-// TODO: Remove this FIX_WHEN_MIN_IS_193
-tasks.findByName("compileIntegrationTestKotlin").onlyIf { resolveIdeProfileName() != "2019.2" /* Tests require 2019.3 or higher to build and run */ }
+tasks.findByName("compileIntegrationTestKotlin")
```
[the remark this answers is not in the document]
Comment (hunterwerlla) [reply jetbrains-rider/build.gradle:79]: Yep, removed

File: jetbrains-rider/tst/software/aws/toolkits/jetbrains/services/lambda/completion/DotNetHandlerCompletionTest.kt (hunk 1 above)
Comment (hunterwerlla) [reply jetbrains-rider/tst/software/aws/toolkits/jetbrains/services/lambda/completion/DotNetHandlerCompletionTest.kt:38]: No idea, it has been broken for a long time. I can take a stab at fixing it

**Pull request merged**